.npmignore

```diff
@@ -1,33 +0,0 @@
-# Created by .ignore support plugin (hsz.mobi)
```

Comment:
When using `.npmignore`, you have to explicitly declare everything that should be ignored instead of what should be included. It can also be confusing when using it along with a `.gitignore` as one could think that it would be taken into account while it's not.

An other solution is to use the package.json's `files` property to list what should be published/installed when getting the package. And the fact is, most of the times there are less things to include than to ignore.

But that's a matter of opinion so let me know what you think ;)